.github/workflows/collector-test.yaml

```diff
@@ -0,0 +1,30 @@
+name: Test Collector Chart
+
+on:
+  pull_request:
+    paths: 
+    - 'charts/opentelemetry-collector/**'
+    branches:
+      - main
+
+jobs:
+  collector-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup
+        uses: ./.github/actions/setup
+        with:
+          create-kind-cluster: "true"
+
+      - name: Run make check-examples
+        run: make check-examples
+
+      - name: Run chart-testing (install)
+        run: ct install --charts charts/opentelemetry-collector
```

Comment:
Why splitting this instead of keeping it in lint-test?

Reply:
It could totally stay in lint-test, but it felt strange to do all the linting and some testing in one workflow, and then do other testing in different workflows. I didn't love how the tests (CI and custom) were split between different workflows. 

So I made lint-test into just lint, and move the test into their own workflows. Now if we need to add tests for collector or operator there is 1 specific workflow for each chart that focuses solely on tests.

Reply:
What about another workflow for test, so we have:
- lint
- test
- collector-extra-test
- operator-extra-test
?

I just don't like copy pasting the it. If we want to add another chart, it has to get the same steps

Reply:
That's a good consideration. I can make that change tomorrow.

Reply:
Ah, I forgot the other reason I change it was because the operator tests need the cert manager installed. By moving it into their own tests I could eliminate the step that checked for changed files and conditionally added the cert manager. 

If we move it into its own workflow I think we'd need to add all the conditional statements back, which I thought we were trying to remove.

Reply:
I could move the cert creation step into the setup action and always create it in the test.yaml workflow, even if it is the collector chart being tested.

Reply:
Ok, if cert-manager is really needed for running `ct install` on the operator, let's keep it as you've done

Reply:
Agreed. I will confirm that dependency and update the PR if it isn't needed. Hopefully it's only needed for the smoke tests 🤞

Reply:
Can confirm, the ct install tests need cert manager:

<img width="1079" alt="image" src="https://user-images.githubusercontent.com/12352919/164489590-d49cb156-dabf-44c6-9afd-aa67533243c9.png">

Reply:
Ok sounds good. Let's keep this approach